File: backend/app/domain/validation/registry.py
from __future__ import annotations

from . import rules
from .types import Validator

DEFAULT_VALIDATORS: list[Validator] = [
    rules.validate_price_presence,
    rules.validate_description_length,
    rules.validate_email_length,
    rules.validate_cta,
    rules.validate_subject,
    rules.validate_placeholders,
]

File: backend/src/domain/validation/registry.ts
import {
  validateCta,
  validateDescriptionLength,
  validateEmailLength,
  validateFeatureCoverage,
  validateForbiddenClaims,
  validatePlaceholders,
  validatePricePresence,
  validateSubject,
} from "./rules.ts";
import type { Validator } from "./types.ts";

export const DEFAULT_VALIDATORS: Validator[] = [
  validatePricePresence,
  validateDescriptionLength,
  validateEmailLength,
  validateCta,
  validateSubject,
  validateFeatureCoverage,
  validatePlaceholders,
  validateForbiddenClaims,
];
